graphql/resolvers.go

```diff
@@ -189,6 +189,18 @@ func (r *volumeResolver) Host(ctx context.Context, obj *restModel.APIVolume) (*r
 	return &apiHost, nil
 }
 
+func (r *queryResolver) HasVersion(ctx context.Context, id string) (bool, error) {
+	// We do not check the error here because we only want to return an error if the id is not
+	// a valid patch or version id.
+	if v, _ := r.sc.FindVersionById(id); v != nil {
```

Comment:
Can we case on the error? If it's not StatusNotFound we should still error (or at least log an error)